pkg/api/customization/catalog/validator.go

```diff
@@ -38,3 +45,10 @@ func validateURL(pathURL string) error {
 	}
 	return nil
 }
+
+func validateHelm3Version(helmVersion string) error {
+	if common.IsHelm3(helmVersion) {
+		return nil
+	}
+	return httperror.NewAPIError(httperror.InvalidBodyContent, "Invalid helm 3 version. 'v3' or 'helm_v3' are valid values")
```

Comment:
Just return an error, the caller should decide what to do with it.

Reply:
change error message verbiage